pandemic/record.go

```diff
@@ -68,3 +71,28 @@ func (c CityDeck) probabilityOfEpidemic() float64 {
 		return 0
 	}
 }
+
+func (gs GameState) ProbabilityOfCity(cn string) float64 {
```

Comment:
So we'll probably want to add a `TopOfDeck` section for cards returned from the `Drawn` section after an epidemic. Also there's some conditional probability we'll want to add given the possibility of an outbreak.